feature-configs/deploy/s2i/post_deploy.sh

```diff
@@ -8,7 +8,7 @@ else
     build_status=$(oc get build -n dpdk $last_build -o json | jq '.status.phase' | tr -d '"')
     if [ $build_status == "Complete" ]; then
         exit 0
-    elif [ $build_status == "Running" ]; then
+    elif [ $build_status == "Running" ] || [ $build_status == "Pending" ]; then
```

Comment:
nit: also missing `New` and `Cancelled`. What about putting `exit 1` at the end of the script to catch all unwanted build status?

https://docs.okd.io/4.9/rest_api/workloads_apis/build-build-openshift-io-v1.html#specification

Reply:
nice! i didn't know there more statuses. thanks! i will add it